bot/assets/allowed_filetypes.txt

```diff
@@ -0,0 +1,28 @@
+# videos
+.3gp
+.3g2
+.avi
+.bmp
+.m4v
+.mkv
+.mov
+.mp4
+.mpeg
+.mpg
+.tiff
+.wmv
+.gif
+# Photos
```

Comment:
Doesn't these comments get read by file reader?

Reply:
No, there is a handler especially for that https://github.com/The-Codin-Hole/HotWired-Bot/blob/0a45e395fa0b8efab8548734d2ad5762de4b815c/bot/cogs/malware_protection.py#L22-L26